notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -220,6 +220,56 @@ father and son [Sherman Klump had a heart to heart talk with his father Cletus a
+
+:: Major Themes
+the human capacity for good and evil [Central to the story was the already narcissistic scientist Sebastian Caine losing all sense of morality once he became invisible and didn't need to worry about facing any consequences for his actions anymore.]
+romantic jealousy [Sebastian became increasingly consumed with jealousy after finding out that his colleague Matt was carrying on with Sebastian's old flame, Linda, behind his back.]
+old flames [Sebastian made continued efforts to rekindle his relationship with his old flame and current colleague, Linda, but she had already moved on.]
+office romance [The co-workers at the lab Linda and Matt were having a relationship in secret behind the lab principal investigator Sebastian's back.]
+romantic love [The main characters Linda and Matt were head over heels in love.]
+narcissism [The scientist Sebastian Caine had an over-inflated ego, routinely boasted to others about how much of a genius he was, and even referred to himself as God on two separate occasions.]
+the scientist profession [The viewer is shown a stereotyped view of what it might be like for a team of scientists to work on developing an invisibility serum in a government lab.]
+unrequited love [Central to the story was Sebastian not being able to come to terms with the fact that his ex-girlfriend, Linda, wasn't into him anymore and had moved on to be with somebody else.]
+
+:: Minor Themes
+chemicals [Sebastian's engineering of the novel "Serial Protein Caine 127" protein was integral to the invisibility serum.]
+playing god with nature [Frank chastised Sebastian for "disturbing the natural order of things" and went on to warn him that he would be "severely punished for all eternity" for experimenting with making animals invisible.]
+hubris [The scientist Sebastian Caine explicitly referred to himself as God in connection with his pioneering achievements in developing a serum that rendered its subject invisible.]
+pride goes before a fall [The scientist Sebastian Caine dared to think himself as a god after having successfully turned himself invisible, but in the end he fell down an elevator shaft to what one presumes was his fiery death.]
+the desire for fame [A point was made about Sebastian hungering after a Noble Prize in recognition of his pioneering work on an invisibility serum.]
+vivisection [Dr. Sarah Kennedy called Sebastian an "unethical bastard" for his planning to vivisect a gorilla.]
+risk taking vs. playing it safe [The lead scientists on the team weighed the risks of proceeding to test the invisibility serum on humans before the animal testing was complete owing to a hard deadline imposed by the military. They ultimately resolved to proceed to human testing.]
+nonconsensual voyeurism [An comfortably invisible Sebastian was peeping on Linda and other women.]
+coping with unwelcome romantic attention [Sebastian made a variety of creepy advances on his old flame Linda.]
```

Comment:
Comment doesn't explain that she found them unwelcome though (I usually don't point this out as it's easy to read between the line, but the comment would be better if it could be like "Linda was perturbed by her ex Sebastian's creepy advances.")

Reply:
Good call. I updated the comment to a slight variation on your suggestion.